### DISCUSSION TITLE
标题出错

### DISCUSSION THREAD

**Discussion by xiaoxiaoxiaoxiaocai** [valor-x/hexo-theme-solitude#315] [General]
更换渲染器后，三级前的标题的最后面出现点，之后的就在标题前出现了点。
![屏幕截图 2024-06-25 113706](https://github.com/everfu/hexo-theme-solitude/assets/131463449/802bad4d-b344-48b0-b109-9d7549748e6d)

**Comment by xiaoxiaoxiaoxiaocai** [comment]
原因是什么

**Reply by everfu** [reply]
你没配置好，可以看看我博客写的更换 markdown-it 的文章，看看我和你的不同

**Comment by xiaoxiaoxiaoxiaocai** [comment]
![屏幕截图 2024-06-27 100703](https://github.com/everfu/hexo-theme-solitude/assets/131463449/21d86f0e-5e02-466a-885c-45b784f54d82)
不知道为什么

**Comment by xiaoxiaoxiaoxiaocai** [comment]
我只要加上数学公式就标题出错，删掉plugins的数学公式就没事

**Reply by everfu** [reply]
你安装数学公式插件了吗

**Reply by xiaoxiaoxiaoxiaocai** [reply]
安装了，数学公式能用，但是就是标题会出错

**Reply by xiaoxiaoxiaoxiaocai** [reply]
![屏幕截图 2024-06-28 104247](https://github.com/everfu/hexo-theme-solitude/assets/131463449/25a0bea2-5ebc-405d-bc55-74c7cd67e96a)

![屏幕截图 2024-06-28 104233](https://github.com/everfu/hexo-theme-solitude/assets/131463449/ae2a9cb2-1dbb-4504-8d55-c410e8dab701)

**Comment by xiaoxiaoxiaoxiaocai** [comment]
我只要加上数学公式就标题出错，删掉plugins的数学公式就没事
